lib/sinon/util/core/is-es-module-getter.js

```diff
@@ -0,0 +1,24 @@
+"use-strict";
+
+const getPropertyDescriptor = require("./get-property-descriptor");
+
+/**
+* Returns `true` if the object is an ES Module and the property is a getter.
+* This is used to support exports as generated by `swc`.
+*
+* @param object the object to examine 
+* @param property the property to examine
+*
+* @returns {boolean}
+*/
+
+module.exports = function isESModuleGetter(object, property) {
+  const descriptor = getPropertyDescriptor(object, property);
+  return (
+    object &&
+    // eslint-disable-next-line no-underscore-dangle
+    object.__esModule &&
```

Comment:
Would you happen to know if `__esModule` is more or less a standard shared between the various transpilers (Webpack, SWC, etc)?